appstore-metadata-service/src/main/resources/static/appstore-metadata-service.yaml

```diff
@@ -670,6 +670,10 @@ components:
           type: string
           description: Application type (example&#58; application/vnd.rdk-app.dac.lightning)
           example: application/vnd.rdk-app.dac.native
+        size:
+          type: integer
+          description: Size in bytes
+          example: 10000000
```

Comment:
@djasiok let's validate for minimum/maximum values:
 https://swagger.io/docs/specification/data-models/data-types/#numbers